### DISCUSSION TITLE
Super Tiny Change for Gender based Possessive pronouns

### DISCUSSION THREAD

**Discussion by Aethelred-The-Ready** [mars-sim/mars-sim#328] [General]
Ok this is totally not important, but can we change the possessive pronoun if gender is not know to their? Just feels weird to use its. Though we will lose the nice symmetry of them all being the same length, which might be important for something. I don't currently have the resources to test this extensively, and it being an edge case already means it is even tricker, so if whoever coded it originally could give a hand that would be great

**Comment by mokun** [comment]
@Aethelred-The-Ready ,

> can we change the possessive pronoun if gender is not know to their?

Can you cite a few examples here so that we know for sure what you mean ? 

Appreciate for your support !
